clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -1343,6 +2015,16 @@ RValue CIRGenFunction::buildBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
   }
   }
 
+  // If a Builtin is not handled in above code but handled in original
+  // clang codegen, we will assert here with its name.
+  // It would be confusing and bad for triaging to let it go on
+  // as they will fail evetunally with unrelated message.
+  std::string unhandledName = getUnhandledBuiltinName(BuiltinID);
+  if (!unhandledName.empty()) {
+    llvm::errs() << unhandledName << " ";
+    llvm_unreachable("NYI");
```

Comment:
I think it'll be easier to parse the error if the unreachable message itself includes the builtin name.